src/lib.rs

```diff
@@ -73,11 +75,11 @@ impl Builder {
         self
     }
 
-    pub fn files<T: Into<String> + Clone>(&mut self, files: &[T]) -> &mut Self {
-        self.files = files
-            .iter()
-            .map(|t| t.clone().into())
-            .collect::<Vec<String>>();
+    pub fn files<'a, T: 'a>(&mut self, files: &'a [T]) -> &mut Self
```

Comment:
I don't get it, why an explicit lifetime is necessary?

Reply:
I guess we can use an HRTB instead. You need some kind of lifetime because we are iterating over `&[T]`, which means `t: &T` and if you want to talk about `&T` in a where clause (or bound) you must make the lifetime explicit.